contract_sale_generation/tests/common.py

```diff
@@ -0,0 +1,132 @@
+# © 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# Copyright 2017 Pesol (<http://pesol.es>)
+# Copyright 2017 Angel Moya <angel.moya@pesol.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from freezegun import freeze_time
+
+from odoo import fields
+from odoo.tests import Form
+
+
+def to_date(date):
```

Comment:
This one is not uses and can be removed

Reply:
No, but should be in each test line that uses fields.Date.to_date(